tests/runtests.py

```diff
@@ -243,7 +243,7 @@ def actual_test_processes(parallel):
 
 
 def django_tests(verbosity, interactive, failfast, keepdb, reverse,
-                 test_labels, debug_sql, parallel):
+                 test_labels, debug_sql, parallel, tags=None, exclude_tags=None):
```

Comment:
I don't see a need for these to be kwargs. Can you evaluate the need for that elsewhere too? In general, the uncertainty  between tags/exclude_tags between a set or None seems confusing.